hypervisor/arch/x86/cpu.c

```diff
@@ -579,7 +577,7 @@ void cpu_secondary_init(void)
 	/* Find the logical ID of this CPU given the LAPIC ID
 	 * and Set state for this CPU to initializing
 	 */
-	cpu_set_current_state(cpu_find_logical_id
+	cpu_set_current_state(get_cpu_id_from_lapic_id
 			      (get_cur_lapic_id()),
```

Comment:
can we move this line up to be together with get_cpu_id_from_lapic_id without violating the coding style?